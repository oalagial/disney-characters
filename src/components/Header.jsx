import React from 'react'
import styled from 'styled-components'
import logo from '../assets/disney-icon-big.png'

const Header = () => {
  const HeaderContainer = styled.div`
    width: 100%;
    height: 100px;
    background-color: #0f3b84;
    display: flex;
    justify-content: center;
    align-items: center;
  `

  const HeaderTitle = styled.div`
    margin-top: 18px;
    font-size: 56px;
    color: gold;
    margin-left: 16px;
    marginright: 16px;
    font-family: 'Mouse Memoirs', sans-serif;
  `

  return (
    <HeaderContainer>
      <img src={logo} alt="Logo" />
      <HeaderTitle role="heading">Disney Characters</HeaderTitle>
    </HeaderContainer>
  )
}

export default Header
