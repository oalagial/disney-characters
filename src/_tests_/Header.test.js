import React from 'react'
import renderer from 'react-test-renderer'
import Header from '../components/Header'

describe('Header', () => {
  it('Matches DOM Snapshot', () => {
    const domTree = renderer.create(<Header />).toJSON()
    expect(domTree).toMatchSnapshot()
  })
})
