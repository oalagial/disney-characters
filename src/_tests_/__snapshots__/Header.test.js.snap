// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Matches DOM Snapshot 1`] = `
<div
  className="sc-bcXHqe fTAxBi"
>
  <img
    alt="Logo"
    src="disney-icon-big.png"
  />
  <div
    className="sc-gswNZR bKdiIt"
    role="heading"
  >
    Disney Characters
  </div>
</div>
`;
